dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -22,12 +25,23 @@ import okhttp3.OkHttpClient
  * This interceptor will combine the effects of the [TracingInterceptor] and the
  * [RumInterceptor].
  *
+ * From [RumInterceptor]: this interceptor will log the request as a RUM resource, and fill the
```

Comment:
In other places we refer to RUM Resources (and other RUM types) with a capital letter. So "RUM **R**esource", instead of "RUM resource" if we want to keep it consistent.